proxy/src/transport/tls/dns_name.rs

```diff
@@ -0,0 +1,30 @@
+use super::webpki;
```

Comment:
It's surprising that this is referenced from `super` -- is this because the `extern crate` is limited to the `tls` module?

Reply:
> It's surprising that this is referenced from super -- is this because the extern crate is limited to the tls module?

Yes.